fmi_weather_client/http.py

```diff
@@ -85,19 +88,19 @@ def _create_params(request_type: RequestType,
         raise ValueError("Missing location parameter")
 
     if request_type is RequestType.WEATHER:
-        end_time = datetime.utcnow().replace(tzinfo=timezone.utc)
+        end_time = datetime.now(UTC)
```

Comment:
`datetime.now` is fine, but please use `timezone.utc` as a parameter instead to keep backward compatibility.